include/openrave/kinbody.h

```diff
@@ -1310,7 +1317,8 @@ class OPENRAVE_API KinBody : public InterfaceBase
         ///
         /// the list is stored inside _GetInfo()._mapExtraGeometries. Note that the pointers are copied and not the data, so
         /// any be careful not to modify the geometries afterwards
-        void SetGroupGeometries(const std::string& name, const std::vector<KinBody::GeometryInfoPtr>& geometries);
+        /// \param deferPostprocessChangedParameters If true, this function will not post a Prop_LinkGeometryGroup update to the body. It will be up to the caller to ensure this occurs.
+        void SetGroupGeometries(const std::string& name, const std::vector<KinBody::GeometryInfoPtr>& geometries, bool deferPostprocessChangedParameters = false);
```

Comment:
seems you are only using the `deferPostprocessChangedParameters` for the internal computations, in which case i would create a different private function that skips the postprocessing, and leave the public function as is.